opensearch/examples/json.rs

```diff
@@ -0,0 +1,87 @@
+use serde_json::json;
+
+use opensearch::auth::Credentials;
+use opensearch::cert::CertificateValidation;
+use opensearch::http::headers::HeaderMap;
+use opensearch::http::transport::{SingleNodeConnectionPool, TransportBuilder};
+use opensearch::http::{Method, Url};
+use opensearch::OpenSearch;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let url = Url::parse("https://localhost:9200")?;
+    let credentials = Credentials::Basic("admin".into(), "admin".into());
+    let transport = TransportBuilder::new(SingleNodeConnectionPool::new(url))
+        .cert_validation(CertificateValidation::None)
+        .auth(credentials)
+        .build()?;
+    let client = OpenSearch::new(transport);
+
+    // Delete Index
+    let response = client
+        .send(
+            Method::Delete,
+            "/movies",
+            HeaderMap::new(),
+            Option::<&String>::None,
+            Option::<&String>::None,
+            None,
+        )
+        .await?;
```

Comment:
Check whether the index exists, then delete it. Or just remove this as the sample deletes the index at the end.